allensdk/brain_observatory/ecephys/ecephys_project_cache.py

```diff
@@ -684,12 +685,15 @@ def from_warehouse(cls,
         fetch_tries : int
             Maximum number of times to attempt a download before giving up and
             raising an exception. Note that this is total tries, not retries
+        timeout
+            Timeout for the RmaEngine constructor. Defaults to 50 minutes.
```

Comment:
I would be a bit more specific here with:

```
timeout : int
    Amount of time (in seconds) to wait on an HTTP request before raising an error.
    Increase this duration if you find that warehouse servers are not responding quickly.
    Defaults to 3000 seconds (50 minutes).
```

Reply:
Done.